src/tracks/SampleTrack.cpp

```diff
@@ -280,9 +280,9 @@ void SampleTCO::saveSettings( QDomDocument & _doc, QDomElement & _this )
 	{
 		_this.setAttribute( "color", color().name() );
 	}
-	if(m_sampleBuffer->reversed())
+	if (m_sampleBuffer->reversed())
 	{
-		_this.setAttribute("reversed", "true");
+		_this.setAttribute ("reversed", "true");
```

Comment:
```suggestion
		_this.setAttribute("reversed", "true");
```

Reply:
Can you clarify on whether the opening brackets should have a space before them or not? You seem to have used different formats in different spaces; I'd like to keep things uniform now that we're changing the style.

Reply:
This is a function call, there should be no space between the function name and the opening parenthesis. The only time there is a space before one is when it's preceded by a keyword (`if`, `for`, etc.).

What snippet did I make that added a space?

Reply:
> The only time there is a space before one is when it's preceded by a keyword (if, for, etc.).

Ah, didn't know that was the case, I thought it was supposed to be uniform across all. Thanks for the clarification!